Project/ProjectSettings/NetworkManager.asset

```diff
@@ -0,0 +1,8 @@
+%YAML 1.1
```

Comment:
Why do we need this `.asset` now ?

Reply:
maybe an artifact. not intentionally added. will remove

Reply:
removed